apps/asap-server/test/fixtures/events.fixtures.ts

```diff
@@ -1,172 +1,38 @@
-                flatData: {
-                  googleCalendarId:
-                    'c_v8ma9lsbsjf90rk30ougr54iig@group.calendar.google.com',
-                  color: '#7A367A',
-                  name: 'Tech 3 - Structural Biology',
-                },
-                referencingGroupsContents: [
-                  queryGroupsResponse.data.queryGroupsContentsWithTotal
-                    .items[0],
-                ],
-              },
-            ],
-          },
-        },
-      ],
-    },
-  },
-};
-
-export const listEventResponse: ListEventResponse = {
-  total: 2,
-  items: [
-    {
-      id: 'afcee0ec-fcd5-479c-9809-e397636f815a',
-      title: 'Example Event',
-      startDate: '2009-12-02T16:19:31.000Z',
-      startDateTimeZone: 'Europe/London',
-      endDate: '2009-12-24T16:20:14.000Z',
-      endDateTimeZone: 'Europe/London',
-      description: 'This event is awesome',
-      thumbnail: `${config.baseUrl}/api/assets/${config.appName}/uuid-thumbnail-2`,
-      lastModifiedDate: '2021-02-08T16:22:12.000Z',
-      notes: 'These are the notes from the meeting',
-      videoRecording: '<embeded>video</embeded>',
-      presentation: '<embeded>presentation</embeded>',
-      meetingMaterials: [
-        {
-          title: 'My additional link',
-          url: 'https://link.pt/additional-material',
-        },
-      ],
-      meetingLink: 'https://zoom.com/room/123',
-      status: 'Cancelled',
-      tags: ['A'],
-      calendar: {
-        id: 'c_t92qa82jd702q1fkreoi0hf4hk@group.calendar.google.com',
-        color: '#125A12',
-        name: 'Tech 1 - Sequencing/omics',
-      },
-      group: listGroupsResponse.items[0],
-    },
-    {
-      id: 'a820b5b7-8f7a-4297-a5a5-cf48b53ba3f7',
-      title: 'This is another event',
-      description: '',
-      startDate: '2021-02-08T16:20:32.000Z',
-      startDateTimeZone: 'Europe/London',
-      endDate: '2021-02-08T16:20:32.000Z',
-      endDateTimeZone: 'Europe/London',
-      thumbnail: `${config.baseUrl}/api/assets/${config.appName}/uuid-thumbnail-1`,
-      lastModifiedDate: '2021-02-08T16:20:35.000Z',
-      status: 'Confirmed',
-      notes: null,
-      videoRecording: null,
-      presentation: null,
-      meetingMaterials: null,
-      tags: ['B'],
-      calendar: {
-        id: 'c_v8ma9lsbsjf90rk30ougr54iig@group.calendar.google.com',
-        color: '#7A367A',
-        name: 'Tech 3 - Structural Biology',
-      },
-      group: listGroupsResponse.items[0],
-    },
-  ],
+export const eventRestResponse: Event = {
+  googleId: 'google-event-id',
+  title: 'Event Tittle',
+  description: 'This event will be good',
+  startDate: '2021-02-23T19:32:00Z',
+  startDateTimeZone: 'Europe/Lisbon',
+  endDate: '2021-02-23T19:32:00Z',
+  endDateTimeZone: 'Europe/Lisbon',
+  calendar: ['squidex-calendar-id'],
+  status: 'Confirmed' as EventStatus,
+  tags: [],
+  hidden: false,
+  meetingLink: 'https://zweem.com',
 };
 
 export const eventResponse: EventResponse = {
```

Comment:
should these be generators too instead of just consts?

Reply:
good catch, thanks